server/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const db = require('../database/index');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+
+const app = express();
+const PORT = 3002;
+
```

Comment:
are you going to serve up a static file when they access /books/:id endpoint initially?

Reply:
I am going to serve a static file in the index page "/"